playfab-docs/api-references/events/data-types/playerprofile.md

```diff
@@ -24,9 +24,14 @@ ms.localizationpriority: medium
 |DisplayName|String|Player Display Name|
 |LastLogin|DateTime|Last login|
 |LinkedAccounts|[PlayerLinkedAccount](playerlinkedaccount.md)|Array of third party accounts linked to this player|
-|Locations|[PlayerLocation](playerlocation.md)|Dictionary of player's locations by type.|
+|Locations_LastLogin_City|String|City of the player's geographic location on last login|
+|Locations_LastLogin_ContinentCode|[ContinentCode](continentcode.md)|The two-character continent code for this location|
+|Locations_LastLogin_CountryCode|[CountryCode](countrycode.md)|The two-character ISO 3166-1 country code for the country associated with the location|
+|Locations_LastLogin_Latitude|Real|Latitude coordinate of the player's geographic location on last login|
+|Locations_LastLogin_Longitude|Real|Longitude coordinate of the player's geographic location on last login|
 |Origination|[LoginIdentityProvider](loginidentityprovider.md)|Player account origination|
 |PlayerId|String|PlayFab Player ID|
+|PlayerMemberships|Dynamic|Array of memberships this player owns|
```

Comment:
>Dynamic [](start = 19, length = 7)

Hmmm .... PushNotificationRegistrations is also dynamic type but we don't call it out there. 

Is this location for defining Kusto table? Or will this mess with people's understanding of the player profile api

Reply:
This is the Kusto table only. I'll need to mention that in the other docs page. The player profile API is defined here: https://api.playfab.com/documentation/Server/datatype/PlayFab.Server.Models/PlayFab.Server.Models.PlayerProfileModel

We do call out the nested structure underneath this here: https://docs.microsoft.com/en-us/gaming/playfab/api-references/events/data-types/pushnotificationregistration